.env-cmdrc

```diff
@@ -34,6 +34,10 @@
     "CX_B2B": "true",
     "CX_CPQ": "true"
   },
+  "segment-refs": {
+    "CX_BASE_URL": "https://api.cg79x9wuu9-eccommerc1-p7-public.model-t.myhybris.cloud",
```

Comment:
Is it required to checkin our ccv2 api url?

Reply:
this file is not shipped to customers & for internal purpose only. Since Personalization is configured in P7 system, if we run SPA_ENV-segment-refs npm start - it will automatically take P7 instead of the "https://40.76.109.9:9002" to test this feature